src/utils/managing.officer.corporate.ts

```diff
@@ -72,6 +75,14 @@ export const getManagingOfficerCorporateById = (req: Request, res: Response, nex
       templateOptions[StartDateKey] = startDate;
     }
 
+    // Start: Redis removal work - Add extra template options if Redis Remove flag is true - needs refacotoring into a common method
+    const isRegistration: boolean = req.path.startsWith(config.LANDING_URL);
```

Comment:
Rather than do this, can't you use the new function that @markpit wrote for the BO Other change to get the additional values added to the `templateOptions`?

Reply:
Using the new function provided